src/click.js

```diff
@@ -3,7 +3,7 @@ export let context = stars.getContext("2d")
 let paint = document.getElementById("paint");
 let or = paint.getContext("2d")
 
-export function getpixelcolour(e) {
+export default function getPixelColour(e) {
```

Comment:
Why are you use 'export default here'? Explain me pls difference between simple export and export default.